studio/components/interfaces/Database/Extensions/Extensions.tsx

```diff
@@ -13,9 +14,10 @@ const Extensions: FC<Props> = ({}) => {
   const [filterString, setFilterString] = useState<string>('')
 
   const extensions =
-    filterString.length === 0
+    (filterString.length === 0
       ? meta.extensions.list()
```

Comment:
need to filter here as well for hidden extensions, if there's no filterString

Reply:
the `.filter()` should cover both cases, but I've refactored it to make it clearer